src/index.html

```diff
@@ -14,105 +14,142 @@
     <header class="header">
       <h1>riddle me this…</h1>
     </header>
+    <main>
+      <section class="page app-grid">
+        <section class="card" data-js="card">
+          <i class="card__bookmark fa fa-3x fa-bookmark" data-js="bookmark"></i>
+          <h2>
+            How do you make the number 7 even without addition, subtraction,
+            multiplication, or division?
+          </h2>
+          <button class="card__show-answer">Show Answer</button>
+          <p class="card__show-answer__answer text-center hidden">
+            Drop the "s"… 😅
+          </p>
+          <ul class="card__tag-list flex-se">
```

Comment:
Too much connection between the card component and the tag-list. I'd prefer just having `tag-list` here and not mentioning the `card` again as it is not important for the tag-list to be part of the card.

Below you can use `tag-list__item` then, which is correct BEM.